README.md

```diff
@@ -237,34 +237,39 @@ you're more than welcome to participate!
 * [Alan West](https://github.com/alanwest), New Relic
 * [Mikel Blanchard](https://github.com/CodeBlanch), Microsoft
 
+[Emeritus Maintainers](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#emeritus-maintainerapprovertriager):
+
+* [Mike Goldsmith](https://github.com/MikeGoldsmith)
+* [Sergey Kanzhelev](https://github.com/SergeyKanzhelev)
+* [Utkarsh Umesan Pillai](https://github.com/utpilla)
+
```

Comment:
@cijothomas This brings up a question on how to handle people who have changed roles. In your case, you were a maintainer earlier. Would you need to be a part of the Emeritus Maintainers too?

Reply:
Good point! I don't know of a perfect answer. Since I am in approver list now, no need to put in emeritus list. If I move away from approver too, then I can be put in either emeritus roles.